### ISSUE TITLE
Why doesn't E_ResNet have activation functions in the fully-connected layers?

### ISSUE THREAD

**Issue by hbk16** [junyanz/BicycleGAN#9]
I've noticed that `self.fc = nn.Sequential(*[nn.Linear(output_ndf, output_nc)])` or `self.fcVar = nn.Sequential(*[nn.Linear(output_ndf, output_nc)])` is the last layer of E_ResNet, depending on vaeLike is True or not. So why doesn't it need an activation function before being output?

**Comment by junyanz** [comment]
Which activation function do you have in your mind? In either case, the output should be unbounded. It will also be regularized by the VAE loss.  @pathak22
